aikau/src/main/resources/alfresco/services/SearchService.js

```diff
@@ -167,6 +168,7 @@ define(["dojo/_base/declare",
          if (!payload || !payload.term)
          {
             this.alfLog("warn", "A request was made to perform a search but no 'term' attribute was provided", payload, this);
+            this.alfPublish(payload.alfResponseScope + payload.alfResponseTopic + "_FAILURE");
```

Comment:
We should avoid manually adding a scope to a publish, as the resultant payload will pick up the default scope and use that (even if it's correct in this instance). A specific scope should be provided in the call the publish, to ensure consistency of behaviour.

```javascript
this.alfPublish(payload.alfResponseTopic + "_FAILURE", {}, false, false, payload.alfResponseScope);
```

If this appears too unwieldy, the solution would probably be to later update the `alfPublish()` signature.